Content.Server/Atmos/Miasma/RottingSystem.cs

```diff
@@ -42,6 +43,11 @@ public override void Initialize()
         SubscribeLocalEvent<TemperatureComponent, IsRottingEvent>(OnTempIsRotting);
     }
 
+    private void OnPerishableStartup(EntityUid uid, PerishableComponent component, ComponentStartup args)
+    {
+        component.NextPerishUpdate = _timing.CurTime + component.PerishUpdateRate;
+    }
```

Comment:
I think this should be MapInit so it doesn't get bulldozed on save / load of a map.

Reply:
isnt mapinit only called when the map starts

Reply:
@metalgearsloth a

Reply:
Yeah, is it not working on paused stuff a drama? It should just workTM.

Reply:
the problem isnt paused stuff
if its 5 mins into the round, nextupdate being 0 means it will constantly tick and instantly rot after spawning
so nextupdate has to be set when spawning

Reply:
it will be called when its spawned normally on an init-ed as well

Reply:
oh ok